nero_view.py

```diff
@@ -0,0 +1,211 @@
+        self.mph = customtkinter.CTkLabel(
+            master=self.mph_frame, text="N/A", font=customtkinter.CTkFont(size=150, weight="bold"))
+
+        self.mph_label = customtkinter.CTkLabel(
+            master=self.mph_frame, text="mph", font=customtkinter.CTkFont(size=20))
+
+        self.mph_frame.grid(row=0, column=0, sticky="s")
+        self.mph.grid(row=0, column=0)
+        self.mph_label.grid(row=0, column=1, sticky="s")
+
+        self.kph_frame = Frame(self.top_left_frame,
+                               width=550, height=150, bg="black")
+        self.kph = customtkinter.CTkLabel(
+            master=self.kph_frame, text="N/A", font=customtkinter.CTkFont(size=25))
+        self.kph_label = customtkinter.CTkLabel(
+            master=self.kph_frame, text=" kmph", font=customtkinter.CTkFont(size=25))
+
+        self.kph_frame.grid(row=1, column=0, sticky="n")
+        self.kph.grid(row=0, column=0)
+        self.kph_label.grid(row=0, column=1)
+
+        # create top right frame
+        self.status = customtkinter.CTkLabel(
+            master=self.top_right_frame, text="N/A", font=customtkinter.CTkFont(size=100, weight="bold"))
+
+        self.dir = customtkinter.CTkLabel(
+            master=self.top_right_frame, text="N/A", font=customtkinter.CTkFont(size=75, weight="bold"))
+
+        self.status.grid(row=0, column=0, sticky="s")
+        self.dir.grid(row=1, column=0, sticky="n")
+
+        # create bottom left frame
+        self.pack_temp = customtkinter.CTkLabel(
+            master=self.bottom_left_frame, text="N/A", font=customtkinter.CTkFont(size=150, weight="bold"))
+        self.pack_temp_label = customtkinter.CTkLabel(
+            master=self.bottom_left_frame, text="Pack Temperature", font=customtkinter.CTkFont(size=20))
+
+        self.pack_temp.grid(row=0, column=0, sticky="s")
+        self.pack_temp_label.grid(row=1, column=0, sticky="n")
+
+        # create bottom middle frame
+        self.motor_temp = customtkinter.CTkLabel(
+            master=self.bottom_middle_frame, text="N/A", font=customtkinter.CTkFont(size=150, weight="bold"))
+        self.motor_temp_label = customtkinter.CTkLabel(
+            master=self.bottom_middle_frame, text="Motor Temperature", font=customtkinter.CTkFont(size=20))
+
+        self.motor_temp.grid(row=0, column=0, sticky="s")
+        self.motor_temp_label.grid(row=1, column=0, sticky="n")
+
+        # create bottom right frame
+        self.state_charge = customtkinter.CTkLabel(
+            master=self.bottom_right_frame, text="N/A", font=customtkinter.CTkFont(size=150, weight="bold"))
+        self.state_charge_label = customtkinter.CTkLabel(
+            master=self.bottom_right_frame, text="State of Charge", font=customtkinter.CTkFont(size=20))
+
+        self.state_charge.grid(row=0, column=0, sticky="s")
+        self.state_charge_label.grid(row=1, column=0, sticky="n")
+
+        self.check_can()
+        self.update()
+
+    def check_can(self):
```

Comment:
Checking CAN should probably be handled by the controller, right? View should only serve to display the data routed to it by the controller I thought

Reply:
I.e controller should call updateUI() or somethin

Reply:
normally yes but the built in tkinter rerender is too useful to pass up tbh